holidayextras/partner-api-docs
# INT-238 Migrate Partner API docs from Doku Wiki to Github Pages

https://holidayextras.jira.com/browse/INT-238

#### What does this PR do? (please provide any background)

This work takes an old dump of the doku wiki pages and converts them to markdown for github.

#### What tests does this PR have?

None.

#### How can this be tested?

A selection of the files above will need to be proof read to ensure that the doku wiki formatting is correctly converted.

    git clone (repo)
    cd partner-api-docs
    git checkout INT-238
    bundle install
    bundle exec jekyll serve

(Wait 2 mins)

    open http://localhost:4000

#### Any tech debt?

The conversion might not be 100% but there could be diminishing returns in getting a perfect automatic conversion.

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?

![giphy](https://cloud.githubusercontent.com/assets/3158640/25877088/3c773b7a-3519-11e7-976d-d6acabe9d176.gif)

I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

By approving a review you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---
